src/components/common/Extensions/ExtensionItem/ExtensionItem.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { ExtensionItemProps } from './types';
+import ExtensionStatusBadge from '../ExtensionStatusBadge-new/ExtensionStatusBadge';
+import Button from '~shared/Extensions/Button';
+import Icon from '~shared/Icon';
+import { useAsyncFunction, useColonyContext, useExtensionData, useMobile } from '~hooks';
+import { ActionTypes } from '~redux';
+import Link from '~shared/Link/Link';
+import styles from './ExtensionItem.module.css';
+import { isInstalledExtensionData } from '~utils/extensions';
+import { ExtensionStatusBadgeMode } from '../ExtensionStatusBadge-new/types';
+
+const ExtensionItem: FC<ExtensionItemProps> = ({ title, description, version, icon, extensionId }) => {
+  const { formatMessage } = useIntl();
+  const isMobile = useMobile();
+  const { colony } = useColonyContext();
+  const { extensionData } = useExtensionData(extensionId ?? '');
+  const [status, setStatus] = useState<ExtensionStatusBadgeMode>();
+  const [badgeMessage, setBadgeMessage] = useState<string>('');
+
+  const values = useMemo(() => {
```

Comment:
name `values` doesn't say anything, please change it

Reply:
changed to `extensionValues`